docs/admin-manual/configuration-macros.rst

```diff
@@ -4452,11 +4461,6 @@ create a private filesystem for the scratch directory for each job.
     A string that names the Linux LVM logical volume for storage 
     for per-job scratch directories.
 
-:macro-def:`STARTD_ENFORCE_DISK_LIMITS[STARTD]`
-    A boolean that defaults to false that controls whether the
-    starter puts a job on hold that fills the per-job filesystem.
-
-
 condor_schedd Configuration File Entries
```

Comment:
The directly above `THINPOOL_NAME` and `THINPOOL_VOLUME_GROUP_NAME` can also be removed since all of this is information was documented earlier in this section and these knobs no longer exist.